frontend/lib/pages/index-page.tsx

```diff
@@ -92,15 +92,16 @@ export default class IndexPage extends React.Component<IndexPageProps> {
               <div className="tile is-child box">
                 <div className="media">
                   <div className="media-left">
```

Comment:
Do you think we still need this `.media-left` div, or the `.media` div that wraps it?  According to Bulma's documentation on the [Media Object](https://bulma.io/documentation/layout/media-object/) it seems like it's only useful for the layout style that involves a thumbnail to the left or right of text content, which we're not using anymore.  It'd be nice to get rid of any unnecessary markup but if you think we still need it then we can keep it.